.github/workflows/run_tests.yaml

```diff
@@ -51,6 +51,8 @@ jobs:
         run: |
           wait-for-it --service 127.0.0.1:7860 -t 600
           python -m pytest -vv --junitxml=test/results.xml --cov . --cov-report=xml --verify-base-url test
+        env:
+          IGNORE_CMD_ARGS_ERRORS: "1"
```

Comment:
The new test will fail without this since a module attempts to read the `pytest` options as regular webui arguments.